MapboxCoreNavigation/Router.swift

```diff
@@ -11,6 +11,10 @@ public protocol RouterDataSource {
 @objc public protocol Router: class, CLLocationManagerDelegate {
     @objc unowned var dataSource: RouterDataSource { get }
     @objc var delegate: RouterDelegate? { get set }
+    
+    @objc(initWithRoute:directions:locationManager:)
+    init(along route: Route, directions: Directions, dataSource source: RouterDataSource)
```

Comment:
Given that this initializer is exposed to Objective-C, does it need to be public as well?

Reply:
The `Router` protocol is public so this initializer is public as well despite the missing `public/open`.